Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -59,6 +59,39 @@ class XcodeSpec: QuickSpec {
 			}
 		}
 
+		describe("locateProjectsInDirectory:") {
+			func relativePathsForProjectsInDirectory(directoryURL: NSURL) -> [String] {
+				let result = locateProjectsInDirectory(directoryURL)
+					.map {
+						return $0.fileURL.absoluteString.substringFromIndex(directoryURL.absoluteString.endIndex)
+					}
+					.collect()
+					.first()
+				expect(result?.error).to(beNil())
+				return result?.value ?? []
+			}
+
+			it("should not find anything in the Carthage Subdirectory") {
+				let relativePaths = relativePathsForProjectsInDirectory(directoryURL)
+				expect(relativePaths).toNot(beEmpty())
+				let pathsStartingWithCarthage = relativePaths.filter { return $0.hasPrefix("Carthage/") }
```

Comment:
I'm going to assume that you mean to add a space between the square-brackets on line 71.